wiki/Storyboarding/id.md

```diff
@@ -0,0 +1,15 @@
+---
+stub: true
+---
+
+# Storyboarding
+
+<!-- TODO: Needs to fit in with storyboard scripting somehow -->
+
+<!-- Two important links are wiki/Beatmap_Editor/Design (visual) and wiki/Storyboard_Scripting (manual coding done by KJ) -->
+
+## Membuat sebuah storyboard
+
+[Storyboard Editor](/wiki/Beatmap_Editor/Design) (SBE) terletak pada Tab Desain (shortcut keyboard F2) di dalam Beatmap Editor. Pada dasarnya, SBE memberikan pengguna cara yang mudah untuk mengontrol pergerakan sprite, tetapi hal itu selalu dianggap sebagai kaku dan tidak berguna untuk semua hal yang lebih dari dasar storyboarding. Jika Anda ingin membuat storyboard untuk layar lebar (widescreen), Anda dapat melakukannya melalui dialog pengaturan lagu dan mengecek kotak Widescreen Support pada Tab Desain.
+
+Para pengguna yang berpengalaman dalam pembuatan storyboard sering menggunakan backend code yang dibuat oleh SBE, berjudul [Storyboard Scripting](/wiki/Storyboard_Scripting) (SBS) sebagai gantinya, yang memberikan mereka kontrol lebih banyak. Lihat artikel tersebut untuk detil lebih lanjut.
```

Comment:
```suggestion
Pengguna yang ahli dalam pembuatan storyboard sering kali menggunakan backend code yang dibuat oleh SBE, berjudul [Storyboard Scripting](/wiki/Storyboard_Scripting) (SBS), dikarenakan kode ini menawarkan mereka kontrol lebih banyak. Lihat artikel tersebut untuk detil lebih lanjut.
```

Reply:
`Para` already say as many similar to `pengguna`, would be confused if there double negative